Flaky test 'TestConsumerBalanceTracker_Handles_GrandTotalChanges'

https://gitlab.com/mysteriumnetwork/node/-/jobs/536443285
```
--- FAIL: TestConsumerBalanceTracker_Handles_GrandTotalChanges (0.06s)
     consumer_balance_tracker_test.go:131: 
         	Error Trace:	consumer_balance_tracker_test.go:131
         	Error:      	Condition never satisfied
         	Test:       	TestConsumerBalanceTracker_Handles_GrandTotalChanges
```

--- a/session/pingpong/consumer_balance_tracker_test.go
+++ b/session/pingpong/consumer_balance_tracker_test.go
@@ -36,7 +36,7 @@ var mockMystSCaddress = common.HexToAddress("0x0")
 
 const initialBalance = 100000000
 
-var defaultWaitTime = time.Millisecond * 50
+var defaultWaitTime = time.Millisecond * 1000
 var defaultWaitInterval = time.Millisecond
 
 func TestConsumerBalanceTracker_Fresh_Registration(t *testing.T) {

